### DISCUSSION TITLE
Are saved comments not downloadable?

### DISCUSSION THREAD

**Discussion by asif-mistry** [Serene-Arc/bulk-downloader-for-reddit#365] [Q&A] [answered]
I am trying to download my saved posts and comments, I am seeing a lot of 'is not a submission' warnings. I have found that non-posts saved comments are not being downloaded. 

Is there a setting to enable downloading comments or is this feature not available yet?

Thanks

Code I used for downloading 
`python3 -m bdfr download "D:\Downloads\bdfr" --user me --saved --authenticate -L 25 --no-dupes`

**Comment by Serene-Arc** [comment, marked as answer]
There is a distinction in the BDFR between the `archive` command and the `download` command. The `download` command downloads any linked resources from the given Reddit object. By their definition, comments have no linked resource and cannot be "downloaded" in that sense. 

However, the `archive` command downloads the Reddit objects themselves, including the metadata at the time. This is the command that you want to download comments, as it will download the comment, any replies, and the metadata associated with it, to be saved in one of the formats given in the README.

**Reply by Binocularbath** [reply]
Hello

Whats the best way to view the JSON file once downloaded?

**Comment by asif-mistry** [comment]
So in order to download my saved comments should I use 
`python3 -m bdfr archive "I:\Downloads\bdfr" --user me --saved --authenticate -L 25 --no-dupes --all-comments` command.

**Reply by Serene-Arc** [reply]
That is correct, apart from the `--all-comments`. Including that will download all of *your* comments i.e. every comment that you've made. Exclude that option and it will download everything from your saved list. You'll have to delete any submission entries.
